staging/src/k8s.io/client-go/applyconfigurations/authentication/v1beta1/tokenreview.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by applyconfiguration-gen. DO NOT EDIT.
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	v1 "k8s.io/client-go/applyconfigurations/meta/v1"
+)
+
+// TokenReviewApplyConfiguration represents an declarative configuration of the TokenReview type for use
```

Comment:
Can you apply one of these?

Reply:
Looks like we cannot. I'll double check why this got generated. The basic approach I've taken is to only generate apply configuration types for types reachable from +clientgen annotated types. But somehow this has slipped in.

Reply:
TokenReview has `+genclient:onlyVerbs=create`. I'll disable generation for types that don't can't be updated/patched.

Reply:
Actually, if we can create, then we can apply? I'll look further tomorrow.

Reply:
Fixed by using the same check we use to decide if a apply function is generated, this way we only generate builders that we actually use (instead of generating them for all types that we generated a typed client for). Change to what we generate: https://gist.github.com/jpbetz/9a2332ebecac8c5ba592c1c031bffb13